Mathlib/Algebra/Category/ModuleCat/Presheaf/Sheafify.lean

```diff
@@ -320,11 +320,11 @@ def toSheafify : M₀ ⟶ (restrictScalars α).obj (sheafify α φ).val where
     simpa using (Sheafify.map_smul_eq α φ (α.app _ r₀) (φ.app _ m₀) (𝟙 _)
       r₀ (by aesop) m₀ (by simp)).symm
 
-instance : Presheaf.IsLocallyInjective J (toSheafify α φ).hom :=
-  by dsimp; infer_instance
+instance : Presheaf.IsLocallyInjective J (toSheafify α φ).hom := by
```

Comment:
If you prefer to automate such changes: #13420 has a style linter (with auto-fix) for this - and is looking for review.